Classes/Service/MailService.php

```diff
@@ -109,9 +120,8 @@ protected function fetchFrom()
      * @codeCoverageIgnore
      * @return array
      */
-    protected function fetchSystemFrom()
+    public function fetchSystemFrom()
```

Comment:
It is not really necessary to make this public or is it?

Reply:
@mbrodala it's easier for testing. I wanted to test all the different variants and fallbacks, so I had to mock the method. Since it's only a wrapper for the static MailUtility call, I don't see an issue with this.

Reply:
Just asking to be sure no obsolete changes are added. ;-)